core/lexicon/ru/setting.inc.php

```diff
@@ -634,6 +634,9 @@
 $_lang['setting_resource_tree_node_name'] = 'Поле для названия узла в дереве ресурсов';
 $_lang['setting_resource_tree_node_name_desc'] = 'Укажите поле ресурса, которое будет использоваться в качестве названия узла в дереве ресурсов. По умолчанию поле «pagetitle», любое поле ресурса может быть использовано: «menutitle», «alias», «longtitle», и т.п.';
 
+$_lang['setting_resource_pagetitle_autocomplete'] = 'Включить автозаполнение для поля заголовка';
```

Comment:
It should be removed from the pull request, all strings will be translated via [Crowdin](https://crowdin.com/project/modx-revolution).